### PR TITLE
chore(internal/gapicgen): enable metadata gen

### PR DESCRIPTION
**Pull request by noahdietz**
* add metadata flag to configs
* update gapic-generator-go to v0.18.2
* gen code changes include:
  * generation of gapic_metadata.json
  * removal of stuttering import aliases for `*pb` packages like `structpb`

### CODE REVIEW

File: internal/gapicgen/generator/config.go [hunk 1]
```diff
@@ -42,6 +42,10 @@ type microgenConfig struct {
 	// useful if a client needs to be deprecated, but retained in the repo
 	// metadata.
 	stopGeneration bool
+
+	// metadata is used to toggle generation of the gapic_metadata.json file
+	// for the client library.
+	metadata bool
```
Comment (codyoss) [inline internal/gapicgen/generator/config.go:48]: When is this not set to true? Should we reverse the var so by default it does not need to be set for every bit of config?

Comment (noahdietz) [reply internal/gapicgen/generator/config.go:48]: Good idea. idk of a case where it shouldn't be enabled.

Comment (noahdietz) [reply internal/gapicgen/generator/config.go]: Done!

**Review by codyoss — APPROVED** [review]
LGTM

**Pull request merged**